lib/ssh_scan/api_db.rb

```diff
@@ -0,0 +1,61 @@
+require 'sqlite3'
+
+module SSHScan
+  class APIDatabaseHelper
```

Comment:
What if we just called it SSHScan::Database as the abstraction layer and then provide an SSHScan::Database::SQLite class?  The reason being is that I think SSHScan::Database should not know anything about SQLite so that we can create SSHScan::Database::Postgres, SSHScan::Database:MySQL, SSHScan::Database:Mongo, etc. without changing interfaces.